szperacz/config.ini

```diff
@@ -0,0 +1,2 @@
+[conf1]
+LOG_LEVEL = 2
```

Comment:
Replace `conf1` with something meaningful, like `Logging`, or `Setup`

Please pay attention to the red sign here: all files shall be ended with newline (just empty line)